tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
Remove this file from your PR we shouldn't be making changes to the typescript config file